content/developer/howtos/rdtraining/13_inheritance.rst

```diff
@@ -71,6 +71,12 @@ The decorator :func:`~odoo.api.model` is necessary for the :meth:`~odoo.models.M
 method because the content of the recordset ``self`` is not relevant in the context of creation,
 but it is not necessary for the other CRUD methods.
 
+It is also important to note that even though we can directly override the
+:meth:`~odoo.models.Model.unlink` method, you will almost always want to write a new method with
+the decorator :func:`~odoo.api.ondelete` instead. Methods marked with this decorator will be
+called during ``unlink()`` and avoids some issues that can occur during uninstalling the model's
+module when ``unlink`` is directly overridden.
```

Comment:
@ticodoo `unlink` is written in 3 different fashions. Shouldn't it always have a link to the ORM API page?

Reply:
@AntoineVDV I think it's fine either way. I'll update it so they're all linked and consistently written.